app/overlays/mtl/ctc_overlay.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_COMP_STUBS=n
+CONFIG_COMP_MODULE_ADAPTER=y
+CONFIG_COMP_GOOGLE_CTC_AUDIO_PROCESSING=y
+CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING=n
```

Comment:
Are RTC and CTC mutually exclusive? If they are then Kconfig files should make sure of that always - not just when using this particular .conf file.

If they are not mutually exclusive then why is RTC being turned off here?

Kconfig and .conf issues are always a nightmare to debug and have a CTC.conf file sneakily and silently turning off RTC is a recipe for confusion.

At the very least this line deserves a comment.

Reply:
Actually: is anyone or anything outside Google ever going to test CTC and use this ctc.conf file? If not then please keep it private; it's only 3-4 lines long.

Reply:
No they are not mutually exclusive

Reply:
> No they are not mutually exclusive

So the `CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING=n` here? It's sneaky.

Reply:
BTW why is this new file placed in the `mtl/` subdirectory? It does not look like it has anything MTL-specific from a technical perspective.

Maybe Google wants this only on MTL right now from a "project management" perspective? In this case, `ctc_overlay.conf` should live in some Google-specific subdirectory or even better - in private configuration files.